mongonaut/exceptions.py

```diff
@@ -1,3 +1,3 @@
 class NoMongoAdminSpecified(Exception):
-    """ Called when no MongoAdmin is specified. Unlikely to ever be called."""
+    """ Called when no MongoAdmin is specified. Unlike to ever be called."""
```

Comment:
That should still read, `Unlikely`, or change the sentence.

Reply:
You are right. The sentence was wrong in version 0.2.20.